rmgpy/molecule/group.py

```diff
@@ -806,9 +806,9 @@ def get_order_str(self):
                 values.append('B')
             elif value == 0:
                 values.append('vdW')
-            elif value == 0.1:
+            elif abs(value - 0.1) < 1e-4:
```

Comment:
Why is this needed?

Reply:
Sometimes floating point errors from incrementing bond orders result in values for the fraction bond orders that are not quite 0.1 or not quite 0.05.